transitions/multi_report_alerts.js

```diff
@@ -7,22 +7,22 @@ const vm = require('vm'),
       messages = require('../lib/messages'),
       utils = require('../lib/utils'),
       transitionUtils = require('./utils'),
-      TRANSITION_NAME = 'multi_form_alerts',
+      TRANSITION_NAME = 'multi_report_alerts',
       BATCH_SIZE = 100,
       requiredFields = [
-        'isReportCounted',
+        'is_report_counted',
         'name',
-        'numReportsThreshold',
+        'num_reports_threshold',
         'message',
         'recipients',
-        'timeWindowInDays'
+        'time_window_in_days'
       ];
 
 const getAlertConfig = () => config.get(TRANSITION_NAME);
 
 /* Returned list does not include the change.doc. */
-const fetchReports = (latestTimestamp, timeWindowInDays, formTypes, options) => {
-  return utils.getReportsWithinTimeWindow(latestTimestamp, timeWindowInDays, options)
+const fetchReports = (latestTimestamp, time_window_in_days, formTypes, options) => {
```

Comment:
Don't use snake case in JS - this inconsistency really hurts my brain. It's ok in JSON to be consistent with the rest of the app_settings.